java/adapter/jdbc/pom.xml

```diff
@@ -61,28 +61,24 @@
         <dependency>
             <groupId>com.fasterxml.jackson.dataformat</groupId>
             <artifactId>jackson-dataformat-yaml</artifactId>
```

Comment:
Er, doesn't this let the version float? Is that what we want?

Could we instead depend on jackson-bom here? Since I think that includes all of these four dependencies anyways

Reply:
Jackson BOM define dependency and version used for: core, data format, data types, and an others. For that reason modules only need to add the dependency and the `version` is injected by the jackson bom. In this case jackson-bom add this version for [jackson-dataformat-yaml](https://github.com/FasterXML/jackson-bom/blob/jackson-bom-2.13.2.20220328/pom.xml#L142:L145).

Reply:
That will be the opposite about what jackson-bom was created.

If the doubt is about what jackson dependencies version the project is using could be an option to create jackson variables on the parent for any jackson dependencies needed to make it clear.

Reply:
Ah, okay then. 

In that case shouldn't jackson-bom be under `<dependencyManagement>` in the root pom

Reply:
Yes, this is the second way to use that. Just moved to this section to make more sense.

Reply:
This looks exactly the same if I'm not mistaken

But as long as maven resolves the proper version then I think this is ok, thank you

Reply:
Just a second, it was part of dependencyManagement all the time.

Reply:
…I'm sorry, I totally missed that

Reply:
Looks like this was all my misunderstanding. Thanks for explaining!